File: 프로그래머스/0/120896. 한 번만 등장한 문자/한 번만 등장한 문자.py
def solution(s):
    answer = ''
    count = dict.fromkeys(sorted(s), 0)
    
    for x in s:
        count[x] += 1
    count = {key: value for key, value in count.items() if value == 1 } 

    for key in count:
        answer += key
        
    return answer

File: 프로그래머스/0/120896. 한 번만 등장한 문자/한 번만 등장한 문자.cpp
#include <string>
#include <vector>
#include <map>

using namespace std;

string solution(string s) {
    string answer = "";
    map<char, int> charmap;
    
    for (char c : s){
        charmap[c] += 1;
    }
    
    for (auto it : charmap){
        if (it.second == 1)
            answer.push_back(it.first);
    }
    return answer;
}